app/services/s3_query_service.rb

```diff
@@ -19,31 +19,44 @@ def self.post_curation_config
     configuration.post_curation
   end
 
+  def self.preservation_config
+    configuration.preservation
+  end
+
```

Comment:
This is really the key of this PR. I added a new configuration setting that allows the `S3QueryService` to use a new bucket + region settings that are defined for preservation uses in s3.yml.